controllers/addresspool_controller.go

```diff
@@ -212,3 +230,60 @@ func (r *AddressPoolReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&starlingxv1.AddressPool{}).
 		Complete(r)
 }
+
+// Verify whether we have annotation restore-in-progress
+func (r *AddressPoolReconciler) checkRestoreInProgress(instance *starlingxv1.AddressPool) bool {
+	restoreInProgress, ok := instance.Annotations[cloudManager.RestoreInProgress]
+	if ok && restoreInProgress != "" {
+		return true
+	}
+	return false
+}
+
+// Update status
+func (r *AddressPoolReconciler) RestoreDataNetworkStatus(instance *starlingxv1.AddressPool) error {
```

Comment:
Change the method name to RestoreAddressPoolStatus

Reply:
Updated